### PR TITLE
Update ctype data files to UCD 13.0.0

### PR DESCRIPTION
**Pull request by gouniLee**
### Checklist

* [ ] At least one test case is included for this feature or bug fix.
* [ ] `ReleaseNotes` has added or is not needed.
* [ ] This PR has passed all of the test cases on `Nodejs` and `Chrome Browser`.
* [ ] This PR has passed all of the test cases on `QT/QML`.
* [ ] This is an API breaking change.
* [ ] Requires a major version change.

### Issue Resolved / Feature Added
* Updated ctype localedate to UCD 13.0.0 by running `genctype.js`
* `ctype_c.json  ctype.json  ctype_l.json  ctype_m.json  ctype_n.json  ctype_p.json  ctype_s.json  ctype_z.json` files are updated.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Links
https://www.unicode.org/Public/13.0.0/ucd/
https://www.unicode.org/charts/ : Unicode 13.0 Character Code Charts

### CODE REVIEW

File: js/data/locale/ctype.json [hunk 1]
```diff
@@ -476,12 +494,24 @@
...
+    ],
     "elbasan": [
         [
             66816,
             66863
         ]
     ],
+    "elymaic": [
+        [
+            69600,
```
Comment (ehoogerbeets) [inline js/data/locale/ctype.json]: How about 2 or 3 unit tests to verify these changes are working properly?

Comment (ehoogerbeets) [reply js/data/locale/ctype.json]: Meaning 2 or 3 for the whole PR, not 2 or 3 per file. Just wanted to have a "smoke test" that it is working the way we expect.

**Comment by ehoogerbeets** [comment]
Also, we need a description of the CLDR 36 and UCD 13 updates in the ReleaseNotes.md

File: js/data/locale/ctype.json (hunk 1 above)
Comment (gouniLee) [reply js/data/locale/ctype.json]: Yes, I've added some test cases.

**Comment by gouniLee** [comment]
> Also, we need a description of the CLDR 36 and UCD 13 updates in the ReleaseNotes.md

I updated ilib version number and ReleaseNote in this PR.

**Pull request merged**